src/app/onboarding/templates/templates.page.ts

```diff
@@ -9,12 +11,16 @@ import { PopupService } from '@shared/popup/popup.service';
 })
 export class TemplatesPage implements OnInit {
   templates: [Template];
+  projectIcon: string;
   constructor(
     private service: OnboardingService,
     private popupService: PopupService,
+    public utils: UtilsService,
+    public storage: StorageService
   ) { }
 
   ngOnInit() {
+    this.projectIcon = this.storage.get('selectedProjectIcon');
```

Comment:
this `templates` is always for "industry project"
other type will use `template` route which is totally different